### ISSUE TITLE
Easy output format for e-Services

### ISSUE THREAD

**Issue by jnehring** [freme-project/technical-discussion#89]
The output of FREME is quiet complex due to the use of NIF. We should introduce another output format called "json-simple" that outputs the service without NIF. This issue should introduce the idea and we can discuss if it is useful.

I propose an output like this

```
{
  "plaintext": "Welcome to Berlin!",
  "annotations": [
    {
      "type": "named-entity-annotation",
      "beginIndex": 11,
      "endIndex": 17,
      "entity-spotted": "true",
      "entity-class": "LOCATION",
      "entity-link": "http://dbpedia.org/Berlin",
      "confidence": 0.76
    },
    {
     "type": "terminology-annotation",
      "beginIndex": 11,
      "endIndex": 17,
      "terminology-link": "http://terms.tilde.com/Berlin",
      "confidence": 0.892
    }
  ]
}
```

The features are

* No use of NIF. There the structure of the NIF is way easier.
* We can implement a single converter from NIF to json-simple. It will support the conversion of e-Terminology, e-Translation and e-Entity from NIF to json-simple.
* The goal is to have an easy solution for easy workflows. It will be less powerful then NIF. E.g. when someone sends a turtle with multiple "nif:isString" elements to an e-Service then json-simple just converts one of these to json-simple and drop the rest.

**Comment by xFran** [comment]
This is looking great. Way better idea that I've had in my mind. Please keep me in the loop.
Thank you.

**Comment by koidl** [comment]
Jan perfect! How can we support you best to get this done as soon as possible. I am cautious of the fact that we need to have demos by the end of the year and I don't want to duplicate efforts. Let me know what you think in terms of timeline and how it connects with the demos needed for the deliverables.

**Comment by fsasaki** [comment]
Kevin, all,

can you please wait a bit, at least a day with this? I think you will
otherwise run into a re-engineering effort later.

Thanks,

Felix


2015-11-17 14:50 GMT+01:00 Kevin Koidl <notifications@github.com>:

**Comment by jnehring** [comment]
Alternative solution for the problem that could make the JSON-SIMPLE format obsolete: 93

**Comment by jnehring** [comment]
We implemented the solution in #93
